packages/venia-concept/src/index.css

```diff
@@ -17,6 +17,7 @@
     --venia-text-alt: 117, 117, 117;
     --venia-text-hint: 158, 158, 158;
     --venia-text-spot: 255, 99, 51;
+    --venia-elm: 32, 137, 144;
```

Comment:
We shouldn't need this color. Any CSS declarations with this rule should use `--venia-teal` instead.